controlsfx/src/main/java/org/controlsfx/control/spreadsheet/SpreadsheetCellBase.java

```diff
@@ -1,5 +1,5 @@
 /**
- * Copyright (c) 2013, 2016, 2018 ControlsFX
+ * Copyright (c) 2013, 2016, 2018, 2023s ControlsFX
```

Comment:
same here. also mark the unwanted `s` at the end of 2023

Reply:
I was just following the sequence...,but changed now